prow/cmd/build/controller.go

```diff
@@ -132,6 +154,7 @@ func newController(kc kubernetes.Interface, pjc prowjobset.Interface, pji prowjo
 		recorder:   recorder,
 		totURL:     totURL,
 		workqueue:  rl,
+		aborter:    pjutil.NewProwJobAborter(pjClient, logrus.NewEntry(logrus.StandardLogger())),
```

Comment:
Can we add a field here to identify the logs as from the aborter?

Reply:
Fixed. I added a filed to the logger.